inf_post/src/balanced.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include "balanced.h"
+
+bool balanced(std::string sequance)
```

Comment:
As Tigran said. "Functions which return boolean, their names must start with 'is' "

Reply:
+